plugins/include/cstrike.inc

```diff
@@ -351,6 +351,47 @@ native bool:cs_get_c4_defusing(c4index);
 native cs_set_c4_defusing(c4index, bool:defusing);
 
 /**
+ * Creates an entity using Counter-Strike's custom CreateNamedEntity wrapper.
+ *
+ * @note Unlike other mods CS keeps track of entities using a custom hashtable.
+ *       This function will add entities to this hashtable, providing benefits
+ *       over the default engine functions (used by create_entity() for example):
+ *       - Storing entities in a hashtable allows CS to improve classname lookup
+ *         performance compared to functions like FindEntityByString that usually
```

Comment:
Instead of `FindEntityByString`, it probably would be better to say which amxx function it is (`find_ent_by_class`?)